schemas/deviceToCloud/ground_fix/ground-fix-lte-only-example copy.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
I'd suggest changing the name of the file to something other than ".... example copy.json".

Reply:
Whoops. That slipped under the radar. Will fix.